CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ Release report: TBD
 
 ### Changed
 
+- Fix FIM test `test_remove_audit` to retry to execute install and remove command if fail ([#3562](https://github.com/wazuh/wazuh-qa/pull/3562)) \- (Tests)
```

Comment:
```suggestion
- Improve `test_remove_audit` FIM test to retry install and remove command ([#3562](https://github.com/wazuh/wazuh-qa/pull/3562)) \- (Tests)
```

Reply:
Done in [here](https://github.com/wazuh/wazuh-qa/pull/3562/commits/742a2366754c9e1896f37968eb0f4eb40d91c916)